### PR TITLE
fetch os kernel version in precheck.yml

### LINKED ISSUE #705: Add os kernel version on playbooks/precheck.yml
When creating a cluster, if choosing to use cilium as the network plugin, cilium requires a system kernel version of >= 4.9.17. Therefore, it is necessary to check the kernel version of the nodes in advance during node inspection, so that users can be given more friendly tips when choosing cilium mode.

```yaml
https://github.com/kubean-io/kubean/blob/7638a75694b8ab95f9711a1ef44987c52a83b7d5/playbooks/precheck.yml#L77
```

### PR DESCRIPTION
**Pull request by hangscer8**
**What type of PR is this?**

/kind feature

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #705 

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by hangscer8** [comment]
result:

企业微信截图_6ebcd09d-5e46-4dc9-b22b-86fdb08d8ee4

```
TASK [Show precheck result] ****************************************************************************************************************************************************
ok: [localhost] => {
    "check_result": {
        "node1": {
            "existing_k8s_service": true,
            "os_kernel_version": "3.10.0",
            "os_kernel_version_output": "3.10.0-1160.el7.x86_64",
            "os_type": {
                "ANSI_COLOR": "0;31",
                "BUG_REPORT_URL": "https://bugs.centos.org/",
                "CENTOS_MANTISBT_PROJECT": "CentOS-7",
                "CENTOS_MANTISBT_PROJECT_VERSION": "7",
                "CPE_NAME": "cpe:/o:centos:centos:7",
                "HOME_URL": "https://www.centos.org/",
                "ID": "centos",
                "ID_LIKE": "rhel fedora",
                "NAME": "CentOS Linux",
                "PRETTY_NAME": "CentOS Linux 7 (Core)",
                "REDHAT_SUPPORT_PRODUCT": "centos",
                "REDHAT_SUPPORT_PRODUCT_VERSION": "7",
                "VERSION": "7 (Core)",
                "VERSION_ID": "7"
            },
            "ping_connection": true
        }
    }
}
Thursday 18 May 2023  08:03:43 +0000 (0:00:00.029)       0:00:23.936 **********
```

File: playbooks/precheck.yml [hunk 1]
```diff
@@ -71,6 +72,26 @@
       when:
         - check_existing_k8s_service
 
+    - name: Perform fetch_os_kernel_version
+      shell: bash -c "uname -r"
```
Comment (ErikJiang) [inline playbooks/precheck.yml:76]: https://github.com/kubernetes-sigs/kubespray/blob/master/roles/network_plugin/cilium/tasks/check.yml#L23-L30

You can use the ansible_kernel variable to get the kernel version directly.

**Pull request merged**